Rover/sailboat.cpp

```diff
@@ -141,6 +141,10 @@ void Sailboat::init()
     // sailboat defaults
     if (sail_enabled()) {
         rover.g2.crash_angle.set_default(0);
+
+        // waypoint navigation target should speedup if boat travels faster than WP_SPEED
+        // instead of asking vehicle to slow down which sailboats cannot do
+        rover.g2.wp_nav.enable_overspeed(true);
```

Comment:
This is not always the case, we can be a 100% motoring mode. I think we can just update it in real time?

Reply:
Great, thanks very much for this.  I've changed this to be:

```
        // waypoint navigation target should speedup if boat travels faster than WP_SPEED
        // instead of asking vehicle to slow down which sailboats without motors cannot do
        if (motor_state != UseMotor::USE_MOTOR_ALWAYS) {
            rover.g2.wp_nav.enable_overspeed(true);
        }
```

Reply:
.. and as you say it needs to be updated each iteration.. clearly I need to think about this some more. txs.